### PR TITLE
SiteOrigin Features: Allow Selection of Different Title Header Levels

### LINKED ISSUE #107: SiteOrigin Features: Allow selection of different title header levels
Currently, they default to h5 and users may wish to increase this depending on the importance of the header for SEO purposes.

Reference thread: https://siteorigin.com/thread/how-to-change-siteorigin-features-title-heading/

### PR DESCRIPTION
**Pull request by AlexGStapleton**
This PR resolves #107.

### CODE REVIEW

File: widgets/features/features.php [hunk 1]
```diff
@@ -220,6 +220,20 @@ function get_widget_form(){
 				'default' => false,
 			),
 
+			'title_tag' => array(
+				'type' => 'select',
+				'label' => __( 'Title Text HTML Tag', 'so-widgets-bundle' ),
```
Comment (Misplon) [inline widgets/features/features.php]: @AlexGStapleton SO is using Sentence case so `Title Text HTML Tag` should be changed to `Title text HTML tag`. Personally, I'd prefer Title Case but to match the standard already in use across the plugin and Widgets Bundle, it should be sentence case.

Comment (AlexGStapleton) [reply widgets/features/features.php]: Ah, sorry I had forgotten to check this.

Comment (Misplon) [reply widgets/features/features.php]: No worries, all sorted now.

**Comment by Misplon** [comment]
@gregpriday I'm happy with the functionality. A quick code review when able, thanks mate.

File: widgets/features/tpl/default.php [hunk 2]
```diff
@@ -38,11 +38,11 @@ class="sow-icon-container <?php echo !empty($instance['container_shape']) ? 'sow
 
 				<div class="textwidget">
 					<?php if(!empty($feature['title'])) : ?>
-						<h5>
+						<<?php echo esc_attr( $instance['title_tag'] ); ?>>
```
Comment (gregpriday) [inline widgets/features/tpl/default.php]: I think that `esc_html` might be a better choice here, but both work.

**Pull request merged**